src/signals/incident-management/components/TextInput_b/index.js

```diff
@@ -0,0 +1,43 @@
+// SPDX-License-Identifier: MPL-2.0
+// Copyright (C) 2018 - 2022 Gemeente Amsterdam
```

Comment:
een ideetje om dit meteen in typescript te doen?

Reply:
👍